extension/js/common/browser.ts

```diff
@@ -463,9 +463,14 @@ export class Ui {
         },
       });
     },
-    error: async (text: string): Promise<void> => {
+    error: async (text: string, isHTML: boolean = false): Promise<void> => {
+      if (isHTML) {
```

Comment:
Ternary operator is ok when assigning value or returning, eg
```js
text = isHTML ? Xss.htmlSanitize(text) : Xss.escape(text).replace(/\n/g, '<br>');
```
I only hate ternary operator when it's compounded (hard to read) or when it's not returning any value, eg:
```js
condition ? doBlah() : doBleh();
```
is not good.

But in this case, no prob.

Reply:
Thanks for the proper escaping!

Reply:
Sorry, you know that I sometimes can forget using ternary operators.
Thanks for noticing!